### ISSUE TITLE
Detect lack of system permissions and guide user to correct them

### ISSUE THREAD

**Issue by ifreecarve** [ianyh/Amethyst#700]
## System
Any

## What's the problem?

Amethyst won't work without permissions.  Currently, no action is taken to check whether those permissions have been granted.  The user has to figure out for themselves why Amethyst isn't working.

## How can it be reproduced?

Revoke Accessibility permissions from Amethyst.  Try to use Amethyst.  

See #695

## Possible Solution

Silica can check permissions.  Leverage this feature somehow to alert the user, and possibly open the system preferences dialog for them.

**Comment by spencersteers** [comment]
Were you thinking something along the lines of what Spectacle does?

![image](https://user-images.githubusercontent.com/2586501/40393259-f0cbc4d8-5ddb-11e8-8054-c898324ee1d9.png)

**Comment by ianyh** [comment]
Yeah. I hadn’t figured out how it would look, but that’s a great example.
